src/platform/plugins/shared/discover/public/context_awareness/profiles/document_profile.ts

```diff
@@ -20,6 +20,7 @@ import type { DataSourceContext } from './data_source_profile';
 export enum DocumentType {
   Log = 'log',
   Trace = 'trace',
+  Generic = 'generic',
```

Comment:
`Generic` and `Default` are pretty similar. Is there really a distinction here or would it make sense to just leave it as `Default`?